opentutor_classifier/tests/test_train_classifier.py

```diff
@@ -172,6 +173,49 @@ def test_train_and_predict(
 @pytest.mark.parametrize(
     "lesson,arch,evaluate_inputs,expected_training_result,expected_evaluate_results",
     [
+        (
```

Comment:
let's not add any additional tests with the expected results hard coded python, instead create a [test.csv](https://github.com/opentutor/opentutor-classifier/blob/multi_arch_test/opentutor_classifier/tests/fixtures/data/question1/test.csv) for your example and then add it to [this pytest test](https://github.com/opentutor/opentutor-classifier/blob/ed74708b89eaf553fd4a1bf0d5c1b00e306ced75/opentutor_classifier/tests/test_train_classifier.py#L132)

Reply:
I have added test cases as CSV files. Please check it out.